Frigate Camera Notifications/Stable

```diff
@@ -648,53 +663,54 @@ action:
                     data_template:
                       name: Frigate Notification
                       message: |
-                        DEBUG: 
```

Comment:
In the debug section the trailing whitespace is needed.

Reply:
Necessary perhaps is a bit of a stretch. It's just formatting. 

If people view the debug messages via the trace it's perfectly fine. 

If people view it via the logbook the debug message doesn't respect the new lines so the spaces just make it more legible. That's all.